tools/ci_build/github/azure-pipelines/npm-packaging-pipeline.yml

```diff
@@ -72,12 +73,12 @@ stages:
     timeoutInMinutes: 10
     steps:
     - download: build
-      artifact: 'NPM_packages'
+      artifact: 'onnxruntime-react-native'
```

Comment:
Change to `onnxruntime-react-native` because line 58 is `onnxruntime-react-native`